### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by zsweigart**
Updated gitignore, removed console log, removed token, and updated version

### CODE REVIEW

File: example/components/PlaidLink.tsx [hunk 1]
```diff
@@ -24,9 +24,8 @@ const PlaidComponent = (props: any) => {
                 token: props.token,
             }}
             onSuccess={(success: LinkSuccess) => {
-                console.log("ZDS")
```
Comment (amytang0) [inline example/components/PlaidLink.tsx:27]: heh

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-plaid-link-sdk",
-  "version": "7.0.0-snapshot6",
+  "version": "7.0.0-snapshot7",
```
Comment (amytang0) [inline package.json:3]: are these public btw?

**Review by amytang0 — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (zsweigart) [reply package.json:3]: no they are internal

**Pull request merged**